src/svm/impls/binary/binary.c

```diff
@@ -1549,6 +1708,73 @@ PetscErrorCode SVMGridSearch_Binary(SVM svm)
   PetscFunctionReturn(0);
 }
 
+#undef __FUNCT__
+#define __FUNCT__ "SVMLoadMatGramian_Binary"
+PetscErrorCode SVMLoadMatGramian_Binary(SVM svm,PetscViewer v)
+{
+  MPI_Comm comm;
+  Mat      G;
+
+  PetscFunctionBegin;
+  TRY( PetscObjectGetComm((PetscObject) svm,&comm) );
+
+  /* Create matrix */
+  TRY( MatCreate(comm,&G) );
```

Comment:
use
```
PetscObjectComm((PetscObject) svm)
```
when comm is not argument to several functions

Reply:
Accepted.